scripts/cmake/includedCMakeLists.txt

```diff
@@ -3,21 +3,17 @@
 # the top level of CIME or the top level of ACME. The required
 # CIME_REL_DIR variable is set in CIME to be "." and in ACME
 # to be "cime".  Except for this misdirection, this is meant
-# to be the full-functioning top-level CMakeLists.txt file 
+# to be the full-functioning top-level CMakeLists.txt file
 # [Andy, 10/4/16]
-#  
+#
 
 cmake_minimum_required(VERSION 2.7)
 INCLUDE(ExternalProject)
 
-
-message("Hardwiring include path -- must automate")
-include(/home/asteyer/cime/src/CMake/CIME_initial_setup.cmake)
-
-#include(/home/agsalin/buildmacrocime/Macros.cmake)
-
 PROJECT(E3SM Fortran C)
-message("XXX ${NetCDF_PATH}")
+message("XXX ${NETCDF_PATH}")
```

Comment:
Including the Macros.cmake file gives many of the variables you are hardcoding here.